katello/katello-backup

```diff
@@ -44,14 +44,22 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
+def run_cmd(command)
+  `#{command}`
+  if $?.exitstatus != 0
```

Comment:
Your branch has what @cfouant and I were discussing as a possible alternative form.